### ISSUE TITLE
ACC information for h02 protocol

### ISSUE THREAD

**Issue by santosharjun** [tananaev/traccar#1555]
Does traccar support ACC ignition On/OFF for h02 protocol?

**Comment by tananaev** [comment]
Yes, it does.

**Comment by santosharjun** [comment]
I am not able to see it on OpenGTS web

**Comment by tananaev** [comment]
Are you using config from <a href="https://www.traccar.org/opengts/">here</a> to integrate Traccar with OpenGTS? If yes, that config doesn't support extra parameters like ignition status. You need to modify it.

**Comment by santosharjun** [comment]
Yes, can you please help with how to modify it?

**Comment by tananaev** [comment]
Are you familiar with SQL? You need to find out how to save ignition into the OpenGTS database.

**Comment by santosharjun** [comment]
Yeah, our team is familiar with SQL. Can you let me know how do we give the input data to SQL query? how do we know the field names etc.

**Comment by tananaev** [comment]
In Traccar all additional data is stored in `attributes` column in XML or JSON format. If your device sends it, there should be `ignition` value there.

**Comment by cispune** [comment]
my device send this parameter in but not able to find acc parameter in that ,
{"sat":7,"mcc":404,"mnc":0,"lac":10094,"cell":5701,"index":52,"ip":"111.111.111.111"}
but device support acc as i already tested it.

**Comment by santosharjun** [comment]
"{""alarm"":true,""ignition"":false,""power"":6,""gsm"":4,""index"":1475,""ip"":""106.220.37.228""}"
ignition value is present in alarm packets at regular intervals but how to find the electricity connected/disconnected bit(power cut alarm)?

**Comment by tananaev** [comment]
@cispune what version of Traccar are you using? Also, I need HEX message sample from log.
